### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by zmiklank**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
 # testing-farm-as-github-action
 
-Testing Farm as GitHub Action is a GitHub Action to execute tests by Testing Farm and update Pull Request status.
+Testing Farm as GitHub Action is a GitHub Action for executing tests on the Testing Farm infrastructure.
 
-This action runs on [composite](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action).
-Before a calling this GitHub Action, you must first clone your project by e.g. 
-by GitHub Action [Checkout V2](https://github.com/actions/checkout).
+The tests to run are to be described with a [`fmf` plan](https://tmt.readthedocs.io/en/latest/spec.html) by the user of this GitHub Action.
+Pull Request status is automatically updated after the tests are executed.
```
Comment (phracek) [inline README.md:6]: This is true but can be changed by setting value `update_pull_request_status` to false.

File: README.md [hunk 2]
```diff
@@ -70,6 +71,6 @@ jobs:
           pull_request_status_name: "CentOS 7"
 ```
 
-and as soon as the job is finished, then you will see in pull request status with proper state like:
+and as soon as the job is finished you will see the test results in the pull request:
```
Comment (phracek) [inline README.md:74]:
```suggestion
and as soon as the job is finished you will see the test results in the pull request status:
```

**Review by phracek — CHANGES_REQUESTED** [review]
Only some minor issues. Otherwise LGTM.

File: README.md (hunk 1 above)
Comment (zmiklank) [reply README.md:6]: Thanks for catching this, I will correct it.

**Comment by zmiklank** [comment]
Mentioned issues are fixed now.

**Review by phracek — APPROVED** [review]

**Pull request merged**